.github/terraform/scripts/start_aws_hazelcast_management_center.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -x
+
+MANCENTER_VERSION=$1
+REGION=$2
+TAG_KEY=$3
+TAG_VALUE=$4
+
+mkdir -p ${HOME}/lib
+mkdir -p ${HOME}/logs
+mkdir -p ${HOME}/man
+
+LOG_DIR=${HOME}/logs
+MAN_CENTER_HOME=${HOME}/man
+
+MANCENTER_JAR_URL=https://download.hazelcast.com/management-center/hazelcast-management-center-${MANCENTER_VERSION}.zip
+
+pushd ${HOME}/lib
+    echo "Downloading JAR..."
+    if wget -q "$MANCENTER_JAR_URL"; then
+        echo "Hazelcast Management JAR downloaded successfully."
+    else
+        echo "Hazelcast Management JAR could NOT be downloaded!"
+        exit 1;
+    fi
+    unzip hazelcast-management-center-${MANCENTER_VERSION}.zip
+    cp hazelcast-management-center-${MANCENTER_VERSION}/* ./
+popd
+
+
+sed -i -e "s/REGION/${REGION}/g" ${HOME}/hazelcast-client.yaml
+sed -i -e "s/TAG-KEY/${TAG_KEY}/g" ${HOME}/hazelcast-client.yaml
+sed -i -e "s/TAG-VALUE/${TAG_VALUE}/g" ${HOME}/hazelcast-client.yaml
+
+ 
+java -cp ${HOME}/lib/hazelcast-management-center-${MANCENTER_VERSION}.jar com.hazelcast.webmonitor.cli.MCConfCommandLine  cluster add -H ${MAN_CENTER_HOME} --client-config ${HOME}/hazelcast-client.yaml \
```

Comment:
I'm not sure I get 100% what's going on here. Can you please explain what this line exactly performs?

Reply:
It adds a cluster for Management Center to monitor and specifies the configuration to use by `--client-config`. The data in hazelcast-client.yaml is written in `{MAN_CENTER_HOME}/sql/mc.mv.db`. Management center uses that configuration data to find the cluster when we start it.